config/admins.txt

```diff
@@ -73,6 +73,10 @@ Cuboos = Game Master
 thunder12345 = Game Master
 wjohnston = Game Master
 mandurrh = Game Master
+<<<<<<< HEAD
+dannno = Game Master
+=======
 thurgatar = Game Master
 xerux = Game Master
 dannno = Game Master
+>>>>>>> d5950a62a4f4208a1be32a3424765f23174b9db7
```

Comment:
Conflicts weren't fixed properly here. Remove the git marker tags and adjust the contents as needed.